### PR TITLE
Update build.md for TrueOS

### PR DESCRIPTION
**Pull request by ImperatorS79**

### CODE REVIEW

**Comment by ImperatorS79** [comment]
I just followed how the docs is written

**Comment by plata** [comment]
I know. Would appreciate if you could do that (up to know, we did not have so many OS so the structure wasn't an issue yet).

File: docs/_pages/build.md [hunk 1]
```diff
@@ -39,32 +60,36 @@ sudo apt-get install git maven openjdk-8-jdk openjfx libxext6:i386
...
 ```
 
-## Build
 * Clone the repository from GitHub:
 ```
 git clone https://github.com/PhoenicisOrg/phoenicis.git
 ```
 
-* For TrueOS, create a file called FreeBSD.properties inside phoenicis/phoenicis-configuration/src/main/resources/, similar to Linux.properties, just change those two lines :
+* Create a file called FreeBSD.properties inside phoenicis/phoenicis-configuration/src/main/resources/, similar to Linux.properties, just change those two lines :
```
Comment (plata) [inline docs/_pages/build.md]: Put the path, `FreeBSD.properties` and `Linux.properties` in back ticks.

**Comment by plata** [comment]
Do you want to change the chapter title? If not, I will merge this.

**Comment by ImperatorS79** [comment]
You can merge

**Pull request merged**